include/Config_WiFi.h

```diff
@@ -49,12 +58,19 @@ COPYRIGHT (c) 2017-2019 Mike Dunston
 //
 // DEFINE PORT TO USE FOR JMRI WiFi INTERFACE
 //
-#define DCCPP_JMRI_CLIENT_PORT 2560
+#ifdef CONFIG_DCCPP_JMRI_CLIENT_PORT
+  #define DCCPP_JMRI_CLIENT_PORT CONFIG_DCCPP_JMRI_CLIENT_PORT
+#else
+  #define DCCPP_JMRI_CLIENT_PORT 2560
+#endif
 
```

Comment:
This option has been removed from the latest files and will not be used.

Reply:
ok, will remove.